compiler/rustc_middle/src/ty/util.rs

```diff
@@ -929,6 +917,22 @@ impl<'tcx> ty::TyS<'tcx> {
     }
 }
 
+// njn: hmm
```

Comment:
hmm

that's really sus but seems to be frequently used :o going to try and remove that in a separate pr

Reply:
opened #93290